src/ODEs/ODETools/RungeKutta.jl

```diff
@@ -213,12 +214,24 @@ function zero_initial_guess(op::RungeKuttaNonlinearOperator)
   x0
 end
 
-function rhs!(op::RungeKuttaNonlinearOperator, x::AbstractVector)
-  u = x
+function rhs!(op::RungeKuttaStageNonlinearOperator, x::AbstractVector)
+  u = zeros(eltype(x),length(x))
+  for j in 1:op.i
+    @. u = u + op.a[op.i,j] * op.ui[j]
+  end
+  v = op.vi
+  @. v = (x-op.u0)/(op.dt)
+  rhs!(op.rhs,op.odeop,op.ti,(u,v),op.ode_cache)
+end
+
+function rhs!(op::RungeKuttaUpdateNonlinearOperator, x::AbstractVector)
+  u = zeros(eltype(x),length(x))
```

Comment:
same

Reply:
clean version does not edit RungeKutta.jl